spring-boot-admin-client/src/test/java/de/codecentric/boot/admin/client/config/SpringBootAdminClientAutoConfigurationTest.java

```diff
@@ -76,4 +82,44 @@ public void reactiveEnvironment() {
 				.run((context) -> assertThat(context).hasSingleBean(ApplicationRegistrator.class));
 	}
 
+	@Test
+	public void customBlockingClientInReactiveEnvironment() {
+		ReactiveWebApplicationContextRunner reactiveContextRunner = new ReactiveWebApplicationContextRunner()
+				.withConfiguration(UserConfigurations.of(CustomBlockingConfiguration.class)).withConfiguration(
+						AutoConfigurations.of(EndpointAutoConfiguration.class, WebEndpointAutoConfiguration.class,
+								WebClientAutoConfiguration.class, SpringBootAdminClientAutoConfiguration.class));
+		reactiveContextRunner.withPropertyValues("spring.boot.admin.client.url:http://localhost:8081")
+				.run((context) -> {
+					assertThat(context).hasSingleBean(ApplicationRegistrator.class);
+					assertThat(context).getBean("registrationClient")
+							.isEqualTo(context.getBean(CustomBlockingConfiguration.class).registrationClient);
+				});
+	}
+
+	@Test
+	public void customBlockingClientInBlockingEnvironment() {
+		ReactiveWebApplicationContextRunner reactiveContextRunner = new ReactiveWebApplicationContextRunner()
```

Comment:
This is still reactive environment ;)

Is working anyways, so we will merge and fix the test to prevent another one-week-roundtrip :)